src/styles/_App.scss

```diff
@@ -101,6 +101,14 @@
   margin: 30px;
 }
 
+.spacer-search {
+  margin: 3rem;
+
+  @media screen and (max-width: 992px) {
```

Comment:
Feels like we should have constants for these various max-width values.

BTW, why 991/2? is that a standard somehow. My phone (Samsung S8+) is 768 in portrait mode